phpcs.xml.dist

```diff
@@ -121,6 +119,14 @@
 		<exclude-pattern>*/tests/*</exclude-pattern>
 	</rule>
 
+	<rule ref="Squiz.Commenting.VariableComment.Missing" >
```

Comment:
The following comment makes me believe that we could skip the variable short description when it doesn't add more information: https://github.com/squizlabs/PHP_CodeSniffer/blob/9bcb29cce831a8f9b2c8aee0bba1a57445f9dbf1/src/Standards/Squiz/Sniffs/Commenting/VariableCommentSniff.php#L83

I'm referring to documentation like:

```php
	/**
	 * Instance of PLL_Model.
	 *
	 * @var PLL_Model
	 */
	public $model;
```

Reply:
It seems that you are right. I wonder how it could look in a generated doc.

Reply:
It seems the WP parser doesn't parse the class properties, but I maybe missed something ...